patches/tools-grit-grit-gather-policy_json.py.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/tools/grit/grit/gather/policy_json.py b/tools/grit/grit/gather/policy_json.py
+index 3837d69e1794ba49b36b441f6f5a61b982430621..2b0be13ff344219798b32583b7567afe583fe8b9 100644
+--- a/tools/grit/grit/gather/policy_json.py
++++ b/tools/grit/grit/gather/policy_json.py
+@@ -328,5 +328,6 @@ class PolicyJson(skeleton_gatherer.SkeletonGatherer):
+         'frame_name': 'Google Chrome Frame',
+         'os_name': 'Google Chrome OS',
+       }
++      self._config = {'build': 'brave', 'app_name': 'Brave', 'frame_name': 'Brave Frame', 'os_name': 'Google Chrome OS'}
```

Comment:
this can be overridden via `override_method`.
see an example here: https://github.com/brave/brave-core/blob/a3cf13b829d8b351e2171666379f4fb26bf914c9/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/chrome/desktop_browser_finder.py#L10-L17

you need to create a file to inline and append it with `inline_file_from_src` patch at the end of this file.

Reply:
Resolved in f96a761d6d775d8f3269ab486bec42aa01e29d2e